src/player_activity.cpp

```diff
@@ -137,13 +138,13 @@ void player_activity::do_turn( player *p )
             break;
         case ACT_PICKAXE:
             // Based on speed, not time
-            moves_left -= p->moves;
+            moves_left -= (int)(p->moves * season_mult);
```

Comment:
It's been years since I last cee pee peed this heavily.  What's the difference between (int)(expr) and static_cast<int>(expr)?